include/object/notification.h

```diff
@@ -27,6 +27,11 @@ static inline void maybeReturnSchedContext(notification_t *ntfnPtr, tcb_t *tcb)
     if (sc == tcb->tcbSchedContext) {
         tcb->tcbSchedContext = NULL;
         sc->scTcb = NULL;
+        /* If the current thread returns its sched context then it should not 
```

Comment:
```suggestion
        /* If the current thread returns its sched context then it should not
```

Reply:
It looks like the style checker tried to make this change, but it seems to have failed to do that.